chevah/chevah-keycert
# Iniatial support for PKCS1 and PKCS8 key format.

Scope
====

This adds support for loading RSA and DSA SSH keys from PKCS1 OpenSSL public key PEM format.

Changes
=======

Refactor the X509 cert code and reuse the same parsing for public PEM files.

Add handling for private PKCS8.

Update the demo command to allow loading SSH keys for easy manual testing.


How to test
========

reviewers: @dumol 

This is more a FYI.

----------

Note that chevah-keycert uses a public `paver.sh` file called `brink.sh` and is not using the `chevah-brink` python package.

The `brink.sh` is a bad name, but the idea is that I would like to have a `paver.sh` file which can be used by independent projects, without any other chevah dependencies.

I think that the latest paver.sh is already independent, so the ` brink.sh` from here is obsolete.

------------

To check that it can load different formats I have used:
```
$ ./build/venv/bin/python keycert-demo.py ssh-load-key --file PKCS1_FILE.pem
```
